ivy_tests/test_ivy/test_frontends/test_paddle/test_math.py

```diff
@@ -1998,6 +1998,43 @@ def test_paddle_rsqrt(
     )
 
 
+# scale
+@handle_frontend_test(
+    fn_tree="paddle.scale",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+    ),
+    scale=st.floats(min_value=1, max_value=100),
+    bias=st.floats(min_value=1, max_value=100),
+    bias_after_scale=st.booleans(),
+)
+def test_paddle_scale(
```

Comment:
There are some failing tests related to this function. Can you have a look!

Reply:
a

> There are some failing tests related to this function. Can you have a look!

Yeah ik, the failing test cases are  when I use float values instead of integers while passing to scale or bias arguments but thats what we need to do right according to the scale function. If there is no mistake while defining the logic on the frontend side then is it in the backend side of ivy which is causing the error?

Reply:
That can't be the case because functional API is tested with each backend as well and there seems to be no issues 😄

Reply:
You need to look into your implementation and see if you can find the issue there.